### PR TITLE
Feature/remove hardcoded email addresses

### LINKED ISSUE #319: [Feature Request] Hard coded email adresses
**Problem description**
The server has several hard coded emails. 
* https://github.com/vantage6/vantage6/blob/main/vantage6-server/vantage6/server/templates/mail/reset_password_token.html
* https://github.com/vantage6/vantage6/blob/9d4d043c6069538bbd74543666b4dd873dc283fc/vantage6-server/vantage6/server/resource/recover.py#L190
* `/token/user` 

**Desired solution**
Make this a setting in the configuration file.

### PR DESCRIPTION
**Pull request by bartvanb**
Fix #319 

To repeat question in issue: @frankcorneliusmartin I don't see why you added /token/user in the description of this issue, can you clarify?

### CODE REVIEW

**Comment by frankcorneliusmartin** [comment]
> Fix #319
> 
> To repeat question in issue: @frankcorneliusmartin I don't see why you added /token/user in the description of this issue, can you clarify?

See issue. tl;dr: I dont know

File: vantage6-server/vantage6/server/resource/recover.py [hunk 1]
```diff
@@ -185,14 +186,19 @@ def post(self):
             {"id": str(user.id)}, expires_delta=expires
         )
 
+        email_info = self.config.get("smtp", {})
+        email_sender = email_info.get("username",
+                                      DEFAULT_SUPPORT_EMAIL_ADDRESS)
```
Comment (frankcorneliusmartin) [inline vantage6-server/vantage6/server/resource/recover.py:191]: I wonder if the mail sender is also the support mail address?

**Review by frankcorneliusmartin — COMMENTED** [review]
Looks good however I am not completely sure if the smtp email is also the email you want to use for support in all cases?

File: vantage6-server/vantage6/server/resource/recover.py (hunk 1 above)
Comment (bartvanb) [reply vantage6-server/vantage6/server/resource/recover.py:191]: So you mean that in line 201, we should replace `email_sender` by some settable support email right? We could make an option in the config to set a support email address.

**Review by frankcorneliusmartin — COMMENTED** [review]
See comments

**Review by frankcorneliusmartin — APPROVED** [review]
lgtm

**Pull request merged**